spec/dry_run_spec.rb

```diff
@@ -1,12 +1,11 @@
 require 'spec_helper'
 
 describe Slackistrano do
-  %w[updating reverting updated reverted failed].each do |stage|
+  %w(updating reverting updated reverted failed).each do |stage|
```

Comment:
Thank you for telling us your style. 
I think that makes sense.

Would you mind if I revert this line change? or Would you revert this by yourself?